app/utility/convert-to-pounds.js

```diff
@@ -1,5 +1,5 @@
 const convertToPounds = (valueInPence) => {
-  return (valueInPence / 100).toFixed(2)
+  return (Math.ceil(valueInPence) / 100).toFixed(2)

```

Comment:
When converting to pounds, round up rather than original approach of dropping remainders to more closely mirror DAX